pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+Fixes #PR_NUMBER 
+## Type of change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+## Change Status
+- [ ] Complete, tested, ready to review and merge
+- [ ] Complete, but not tested (may need new tests)
+- [ ] Incomplete/work-in-progress, PR is for discussion/feedback
+## Has This Been Tested
+- [ ] Yes
+- [ ] No
+- [ ] Not necessary
+## Does this require communication between DS and WEB?
```

Comment:
Great addition to the template! Good work, this will help keep us organized and stay well-communicated